pages/common/npm-query.md

```diff
@@ -0,0 +1,32 @@
+# npm query
+
+> Print an array of dependency objects using CSS-like selectors.
+> More information: <https://docs.npmjs.com/cli/v8/commands/npm-query>.
+
+- Print direct dependencies:
+
+`npm query ':root > *'`
+
+- Print all direct production/development dependencies:
+
+`npm query ':root > .{{prod|dev}}'`
+
+- Print dependencies with a specific name:
```

Comment:
I don't exactly understand what this is meant to do.
Please ignore my suggestion if it is wrong.
```suggestion
- Show information about a specific dependency:
```

Reply:
a package could be installed more than one copy based on the semver of the dependencies.